packages/teuchos/doc/index.doc

```diff
@@ -367,7 +367,11 @@ for SPMD parallel programs and useful utilities that depend on these.
     This may be used either by itself, or in combination with
     Teuchos::TimeMonitor for tracking global timer statistics over
     processes.
-  
+    If Teuchos is compiled with <tt>Teuchos_TIME_MASSIF_SNAPSHOTS=ON</tt>
```

Comment:
Can you change this to "If Teuchos is configured with ..."?

Reply:
Done you also need to mention that you need to configure with `Trilinos_ENABLE_Valgrind=ON`?